server/plugin.ts

```diff
@@ -73,6 +74,9 @@ export class WazuhPlugin implements Plugin<WazuhPluginSetup, WazuhPluginStart> {
     let wazuhSecurity;
     core.http.registerRouteHandlerContext('wazuh', async(context, request) => {
       !wazuhSecurity && (wazuhSecurity = await SecurityObj(plugins, context));
+      if(!request.auth.isAuthenticated){
+        disableRequests();
+      }
```

Comment:
**thought**: I'm not sure if this does what you expect. This code is executed on the backend, meanwhile, the request service is executed in the frontend, I don't understand how from the backend you want to communicate to the frontend to disable the requests done from there.